src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -132,43 +132,32 @@ public boolean hasRentalInformation(Client client, RentalInformation rentalInfor
         return addressBook.hasRentalInformation(client, rentalInformation);
     }
 
-    //=========== Filtered Client List Accessors =============================================================
+    //=========== Filtered and Sorted Client List Accessors ==========================================================
 
     /**
      * Returns an unmodifiable view of the list of {@code Client} backed by the internal list of
      * {@code versionedAddressBook}
      */
     @Override
     public ObservableList<Client> getFilteredPersonList() {
-        return filteredClients;
+        return sortedFilteredClients;
```

Comment:
what is the reason for returning sortedFilteredClients instead of filteredClients?

Reply:
It better reflects the property of the list as being both sorted and filtered